service/routes.py

```diff
@@ -182,4 +182,22 @@ def get_item(customer_id, item_id):
         logger.error(f"Customer {customer_id} and corresponding item {item_id} could not be found.")
         abort(status.HTTP_404_NOT_FOUND, f"Customer {customer_id} and corresponding item {item_id} could not be found.")
  
-
+######################################################################
+# DELETE A SHOPCART
+######################################################################
+@app.route("/shopcarts/<int:customer_id>",, methods=["DELETE"])
```

Comment:
There's a double comma here, thats a syntax error...please fix that

Reply:
thank you, I fixed it.